samcli/local/docker/manager.py

```diff
@@ -197,7 +197,7 @@ def _is_rapid_image(image_name: str) -> bool:
         : return bool: True, if the image name ends with rapid-$SAM_CLI_VERSION. False, otherwise
         """
 
-        if not re.search(r":rapid-\d+\.\d+.\d+$", image_name):
+        if not re.search(r":rapid-\d+\.\d+\.[\w\.]+$", image_name):
```

Comment:
nit: Can we add some examples of valid rapid image tags in the docstring?

Reply:
Done.